src/components/createEvent.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import '../css files/cards.css'
+
+class CreateEvent extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            name: '',
+            description: '',
+            startDate: '',
+            endDate: ''
+        }
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleChange(event) {
+        let nam = event.target.name
+        let val = event.target.value
+
+        this.setState({
+            [nam]: val
+        })
+    }
+
+    handleSubmit(event) {
+        // event.preventDefault()
+        fetch('http://127.0.0.1:3001/create-event', {
```

Comment:
This fetch could be transferred to a apiCalls.js file with the others from the app to simplify and separate out some of the logic for encapsulation.